CHANGELOG.md

```diff
@@ -8,6 +8,12 @@ Geometry releases will be tagged as `Descriptive_tag_v_X.Y.Z`.
 
 ## [Unreleased]
 
+##[new_cryostat_window]
```

Comment:
New sections are reserved for new releases. Please just keep you changes mentioned under `## [Unreleased]`.